README.md

```diff
@@ -98,9 +98,25 @@ declaring that, while 4.1.18 and 4.1.20 are in `candidate-4.2`, they should not
 ### Block Edges
 
 Create/edit an appropriate file in `blocked_edges/`.
-- `to` is the release which has the existing incoming edges.
-- `from` is a regex for the previous release versions.
+
+* `to` (required, [string][json-string]) is the release which has the existing incoming edges.
+* `from` (required, [string][json-string]) is a regex for the previous release versions.
     If you want to require `from` to match the full version string (and not just a substring), you must include explicit `^` and `$` anchors.
+    Release version strings will receive [the architecture-suffix](#release-names) before being compared to this regular expression.
+* `url` (optional, [string][json-string]), with a URI documenting the blocking reason.
```

Comment:
IMO justification link should not be optional

Reply:
I think it should, and there's a section in the in-flight enhancement explaining my motivation, but let's hash this out [in the relevant enhancements thread][1].

[1]: https://github.com/openshift/enhancements/pull/821#discussion_r706179172